app/views/api/map/index.xml.builder

```diff
@@ -0,0 +1,8 @@
+xml.instruct!
+
+xml.osm(OSM::API.new.xml_root_attributes) do |osm|
+  osm << (render(:partial => "bounds", :object => @bounds) || "")
+  osm << (render(@nodes) || "")
+  osm << (render(@ways) || "")
+  osm << (render(@relations) || "")
```

Comment:
Why do we need all the `|| ""` stuff here? What does adding a string to the XML even do?

Reply:
That’s a logical or operator. If render returns nil, an empty string is added to the result as a fallback. See https://github.com/openstreetmap/openstreetmap-website/pull/2223#issuecomment-506019631

Reply:
Documentation https://guides.rubyonrails.org/layouts_and_rendering.html says:

> In the event that the collection is empty, render will return nil, so it should be fairly simple to provide alternative content.
> 
> ```
> <h1>Products</h1><%= render(@products) || "There are no products available." %>
> ```